lib/ts/recipe/emailverification/utils.ts

```diff
@@ -68,9 +69,15 @@ export function getEmailVerifyLink(input: {
     token: string;
     recipeId: string;
     tenantId: string;
+    userContext: any;
 }): string {
     return (
-        input.appInfo.websiteDomain.getAsStringDangerous() +
+        input.appInfo
+            .getOrigin({
+                request: getRequestFromUserContext(input.userContext),
```

Comment:
This is not the right way for us to get the request. We should not call `getRequestFromUserContext` here. You need to modify `getEmailVerifyLink` to take in a request object | undefined, and pass that in here. 

Now all the places where `getEmailVerifyLink` is being called, needs to pass in a request object (for example in our APIs). There are a few cases where we don't have the request object like in EmailVerification. createEmailVerificationLink function, or EmailVerification.sendEmailVerificationEmail, but in those cases, you can use `getRequestFromUserContext`.